backpack-common/src/main/java/net/skyscanner/backpack/calendar2/data/CalendarCell.kt

```diff
@@ -98,40 +100,103 @@ internal fun CalendarCellDay(
     yearMonth: YearMonth,
     selection: CalendarSelection,
     params: CalendarParams,
-): CalendarCell.Day = CalendarCell.Day(
-    date = date,
-    yearMonth = yearMonth,
-    info = params.cellsInfo[date] ?: CellInfo.Default,
-    outOfRange = date !in params.range,
-    contentDescription = date.format(params.dateContentDescriptionFormatter),
-    text = buildSpannedString {
-        val span = TtsSpan.DateBuilder()
-            .setDay(date.dayOfMonth)
-            .setMonth(date.month.ordinal)
-            .setYear(date.year)
-            .setWeekday(date.dayOfWeek.value)
-            .build()
-        append(date.dayOfMonth.toString(), span, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
-    },
-    selection = when (selection) {
-        is CalendarSelection.None -> null
-        is CalendarSelection.Single -> when (date) {
-            selection.date -> CalendarCell.Selection.Single
+): CalendarCell.Day {
+    return CalendarCell.Day(
```

Comment:
Can you please use expression body syntax here (instead of return)?

Reply:
Done, thanks!